tools/bin/3d-tiles-tools.js

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
+'use strict';
+var argv = require('yargs').argv;
+var Cesium = require('cesium');
+var fsExtra = require('fs-extra');
+var path = require('path');
+var Promise = require('bluebird');
+var runPipeline = require('../lib/runPipeline');
+
+var fsExtraReadJson = Promise.promisify(fsExtra.readJson);
+
+var defaultValue = Cesium.defaultValue;
+var defined = Cesium.defined;
+
+if (process.argv.length < 4 || defined(argv.h) || defined(argv.help) || !defined(argv._[0])) {
+    var help =
+        'Usage: node ' + path.basename(__filename) + ' command [args]\n' +
```

Comment:
`node` should not be here.  this repository should be installed with `-g` at which point every command would be in the user's system path.